packages/api-generator/src/index.ts

```diff
@@ -63,6 +73,67 @@ const run = async () => {
     }).filter(Boolean)
   )
 
+  // Missing descriptions
+  if (argv.missingDescriptions) {
+    const translations: { [filename in string]?: TranslationData } = {}
+
+    async function readData (filename: string): Promise<TranslationData> {
+      if (!(filename in translations)) {
+        try {
+          const data = JSON.parse(await fs.readFile(filename, 'utf-8'))
+
+          for (const type of ['props', 'events', 'slots', 'exposed']) {
+            for (const item in data[type] ?? {}) {
+              if (data[type][item].startsWith('MISSING DESCRIPTION')) {
+                delete data[type][item]
+              }
+            }
+          }
+
+          translations[filename] = data
+        } catch (e) {
+          translations[filename] = {}
+        }
+      }
+
+      return translations[filename]
+    }
+
+    for (const index in componentData) {
+      const component = componentData[index]
+
+      for (const type of ['props', 'events', 'slots', 'exposed']) {
+        for (const name in component[type]) {
+          if (type === 'props' && !component[type][name].source) {
+            console.warn(`Missing source for ${component.kebabName} ${type}: ${name}`)
+          }
+
+          const filename = type === 'props'
+            ? kebabCase(component[type][name].source ?? componentData[index].componentName)
+            : component.kebabName
+
+          for (const locale of locales) {
+            const sourceData = await readData(`./src/locale/${locale}/${filename}.json`)
+            const githubUrl = `https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/${locale}/${filename}.json`
```

Comment:
Could we possibly do this dynamically so it uses branch that it's run on? Maybe overkill.